src/app/public/modules/autonumeric/autonumeric-options.ts

```diff
@@ -3,10 +3,10 @@ import AutoNumeric, {
 } from 'autonumeric';
 
 /**
- * Specifies custom options to provide directly to the underlying [`autoNumeric` utility](https://github.com/autoNumeric/autoNumeric).
+ * Specifies custom options to provide directly to the underlying [autoNumeric utility](https://github.com/autoNumeric/autoNumeric).
  * The value can be set to `string`, which is an alias that represents
  * a [set of predefined set of options](https://github.com/autoNumeric/autoNumeric#predefined-options)
- * for a currency preset or a language preset, or `Options`, which is a
+ * for a currency or language, or `Options`, which is a
```

Comment:
Just calling out that I couldn't figure out how to add a suggestion to remove "preset" here as well as in the code sample header, so I committed the change directly. Let me know if you think this change is offbase.